ush/set_extrn_mdl_params.sh

```diff
@@ -146,6 +146,13 @@ else
   esac
 
 fi
+#
+#  for retro, the external boundary could come from other location.
+#
+if [[ "${DO_RETRO}" = "true" && ! -z "${EXTRN_MDL_SOURCE_BASEDIR_ICS}" ]]; then
```

Comment:
To make a consistent style, suggest to use double equal sign here: if [[ "${DO_RETRO}" `==` "true" && ! -z "${EXTRN_MDL_SOURCE_BASEDIR_ICS}" ]]; then

Reply:
Done

Reply:
Done